### PR TITLE
Download the Fuchsia SDK and toolchain in a gclient hook.

### PR DESCRIPTION
**Pull request by chinmaygarde**
Downloads into //fuchsia/sdk and //fuchsia/toolchain. //fuchsia is in the buildroot gitignore.

### CODE REVIEW

File: DEPS [hunk 1]
```diff
@@ -441,6 +443,31 @@ deps = {
      'condition': 'download_android_deps',
      'dep_type': 'cipd',
    },
+
+   'src/fuchsia/sdk': {
+     'packages': [
+       {
+        'package': 'fuchsia/sdk/core/${{platform}}',
+        # Get the SDK from https://chrome-infra-packages.appspot.com/p/fuchsia/sdk/core at the 'latest' tag
+        'version': 'jlQvNeRMq6X81_VYiFI_Ol311YCXak0xACebeb8f6TcC'
```
Comment (zanderso) [inline DEPS]: Do mac and linux have the same version string?

Comment (chinmaygarde) [reply DEPS]: Oops. No.

**Comment by chinmaygarde** [comment]
Updated. PTAL.

**Comment by chinmaygarde** [comment]
@pylaligand @Hixie @abarth: What are the licensing implications for pulling in the Fuchsia toolchains & SDK? That's what the license script is failing on ATM.

**Comment by pylaligand** [comment]
There is a LICENSE file at the root of the SDK that you can refer to.

**Pull request merged**